spec/integration/runner/variant-reassignment-example-10.spec.js

```diff
@@ -46,7 +46,7 @@ describe.skip('Variant reassignment', () => {
 
   it('merge variants with different sameForAll attributes without blacklist',
     async () => {
-      const reassignment = new VariantReassignment(ctpClient, logger, {}, ['brandId'])
+      const reassignment = new VariantReassignment(ctpClient, logger, {}, [])
```

Comment:
Just a nitpick - last two parameters can be removed because of default values

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/42/commits/42e89da76e185d4121aeca767e09ab0ab2417f1c